### DISCUSSION TITLE
How to draw a Tracker / Lines at Mouse Position

### DISCUSSION THREAD

**Discussion by arakis** [oxyplot/oxyplot#1762] [Q&A]
What Feature/Flag/Bool needs to be set, to enable the two lines that tracks the mouse position like in this Picture:
![image](https://user-images.githubusercontent.com/1831942/116811914-1c901180-ab4c-11eb-9dd9-b4da73df1e7e.png)
I'm using OxyPlot Avalonia, and have never seen these lines.

**Comment by VisualMelon** [comment]
For WPF and Avalonia, these lines should be configurable by styling the `TrackerControl`. There are some examples here: https://github.com/oxyplot/oxyplot-avalonia/blob/master/Source/Examples/Avalonia/AvaloniaExamples/Examples/CustomTrackerDemo/MainWindow.xaml

However, the lines seem to be broken at the moment in Avalonia. I have opened a PR to fix this: https://github.com/oxyplot/oxyplot-avalonia/pull/29
